deploy/sdk/src/dynamo/sdk/core/protocol/interface.py

```
@@ -74,6 +74,13 @@ class ResourceConfig(BaseModel):
     gpu: str = Field(default="0")
 
 
+class KubernetesOverrides(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)